### DISCUSSION TITLE
TypeError: 'MaskedQuantity' object with a scalar value is not iterable

### DISCUSSION THREAD

**Discussion by Anon-Z1** [lightkurve/lightkurve#1372] [Q&A]
Consistently getting this error code when using lightkurve to normalize lightcurves:
TypeError: 'MaskedQuantity' object with a scalar value is not iterable

![image](https://github.com/lightkurve/lightkurve/assets/147918707/ddf19128-e7e9-49ee-a19c-5946c53f56e6)

Here's the relevant part of the code.
Currently using the latest version of lightkurve

Wondering if anyone else has had this issue/is aware of a solution

**Comment by Nschanche** [comment]
Hi there, could you check out which version of astropy you are using? Upgrading may help this issue (see issue #1268).

**Reply by Anon-Z1** [reply]
Thanks for the quick response

![image](https://github.com/lightkurve/lightkurve/assets/147918707/b8737a75-e316-449d-8f85-4de4e1889152)

Currently using the latest version of astropy

**Comment by Anon-Z1** [comment]
The issue was fixed by updating my version of python
